### ISSUE TITLE
Do the obj of multiple objects need to maintain relative proportional sizes?

### ISSUE THREAD

**Issue by ZhengWangguandong** [NVlabs/FoundationPose#204]
I used bundlesdf to reconstruct the obj files of multiple objects, but they cannot guarantee the relative scale. When applied to the scene_raw point cloud file, the relative scale cannot be maintained, so the bbox and tracking poses are incorrect.
So, 
(1) How can I guarantee the relative scale?
(2) Why can the yellow bottle in the example match the size of scene_raw after being reconstructed using bundlesdf? Is there any trick?

**Comment by wenbowen123** [comment]
BundleSDF can reconstruct metric scale OBJ mesh. What do you mean by relative scale? Are you using the right file?

The mustard bottle is not reconstructed from BundleSDF, but the red driller, book, and snack bos are, which work well.

**Comment by ZhengWangguandong** [comment]
> BundleSDF can reconstruct metric scale OBJ mesh. What do you mean by relative scale? Are you using the right file?
> 
> The mustard bottle is not reconstructed from BundleSDF, but the red driller, book, and snack bos are, which work well.

Thank you for your reply.
I reconstructed multiple objects using bundlesdf, but the relative size ratios between the reconstructed objects are incorrect. For example, I reconstructed the tabletop and table legs. Theoretically, the table legs should be one-eighth the size of the tabletop, but in reality, the table legs are one-quarter the size of the tabletop. This will cause the bbox of the FoundationPose multi-object tracking to be incorrect.
When tracking multiple objects, how can I keep the relative size ratios of the obj of multiple objects correct?
